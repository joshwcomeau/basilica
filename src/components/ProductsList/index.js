// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';
import classNames from 'classnames';

import './index.scss';


const ProductsList = () => {
  const classes = classNames(['products-list']);

  return (
    <div className={classes}>
      Your Component Here :)
    </div>
  );
};

ProductsList.propTypes = {

};

ProductsList.defaultProps = {

};

export default ProductsList;
