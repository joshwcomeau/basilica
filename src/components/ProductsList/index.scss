@import '../../variables';

.products-list {
  position: relative;
  min-height: 1000px;

  @media screen and (max-width: $mq-tablet) {

  }

  @media screen and (max-width: $mq-mobile) {

  }
}
