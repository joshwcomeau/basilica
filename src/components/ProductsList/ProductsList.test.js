/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import ProductsList from './index';

describe('ProductsList', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ProductsList />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
