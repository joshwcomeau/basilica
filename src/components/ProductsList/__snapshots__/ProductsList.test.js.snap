exports[`ProductsList renders correctly 1`] = `
<div
  className="products-list">
  Your Component Here :)
</div>
`;
